code/game/objects/items/kitchen.dm

```diff
@@ -30,6 +30,7 @@
 	hitsound = 'sound/weapons/bladeslice.ogg'
 	armor = list("melee" = 0, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 50, "acid" = 30)
 	var/datum/reagent/forkload //used to eat omelette
+	var/loaded_egg /// True if the loaded item is an egg, false if its pie
```

Comment:
Make it a string instead, not a fucking boolean

Reply:
Changed it